sql/moz-fx-data-shared-prod/amo_prod/desktop_addons_by_client_v1/query.sql

```diff
@@ -57,7 +43,41 @@ WITH per_client AS (
   GROUP BY
     submission_date,
     sample_id,
-    client_id
+    client_id,
+    addons
+),
+per_client AS (
```

Comment:
Unfortunately this is not per_client, it is per_client/app-version/country/locale/app_os. If a user has pings with different values for e.g. country _and_ different `payload.info.addons`, we'll end up with multiple rows for that client in this CTE.

Is there any way we can instead pull this information from `clients_daily_v6`? That table has addon information, and probably already has this other data aggregated.

Reply:
oh you are right ... how about moving that step out and joining it later? 
The problem with `clients_daily_v6` would be that they get the addons data from environment.addons while this query gets it from `payload.info.addons` (which is deprecated as of now) but they values are not the same and so switching would change the outcome ... which we might want to do at some point but maybe not for a triage fix